modules/oadp-backing-up-applications-restic.adoc

```diff
@@ -10,6 +10,11 @@ You back up Kubernetes resources, internal images, and persistent volumes with R
 
 You do not need to specify a snapshot location in the `DataProtectionApplication` CR.
 
+[IMPORTANT]
+====
+Restic does not support backing up `hostPath` volumes. For more information, see link:https://velero.io/docs/v1.9/restic/#limitations[additional Rustic limitations].
```

Comment:
@RichardHoch 
Nowb oth the Velero domain and its version will be attributes, so this link should be: 
https://{velero-domain}/docs/v{velero-version}/restic/#limitations

Reply:
Will change after https://github.com/openshift/openshift-docs/pull/47670 is merged to avoid merge conflicts.

Reply:
@sbeskin-redhat Is PR OK except for the above?

Reply:
@RichardHoch
Yes, I think so. But I would suggest you do the change now and merge the PR after 47670 is merged. Or are you going to merge this without waiting for the OADP GA?

Reply:
@sbeskin-redhat: You convinced to make the change now and to add a note above asking that this PR be merged after PR https://github.com/openshift/openshift-docs/pull/47670.

Reply:
@RichardHoch 
Looks fine